packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2192,35 +2248,56 @@ class AnnotatedRegionLayer<T> extends ContainerLayer {
   /// If not provided, offset defaults to [Offset.zero].
   ///
   /// Ignored if [size] is not set.
+  ///
+  /// Unlike [OffsetLayer], this parameter [offset] does not affect the layer's
+  /// children.
   final Offset offset;
 
-  @override
-  S find<S>(Offset regionOffset) {
-    final S result = super.find<S>(regionOffset);
-    if (result != null)
-      return result;
-    if (size != null && !(offset & size).contains(regionOffset))
-      return null;
-    if (T == S) {
-      final Object untypedResult = value;
-      final S typedResult = untypedResult;
-      return typedResult;
-    }
-    return null;
-  }
+  /// Whether this layer should prevent layers visually behind it from being
```

Comment:
You should turn this comment into a macro with `{@template ...}...{@endtemplate}` (see other macros for an example), and use the `{@macro ...}` on the other `opaque` attributes in this file.